### PR TITLE
Deprecate v0 API

### PR DESCRIPTION
**Pull request by samkim**
* Update v0 references in docs
* ~Remove python 3.6 support~
* Mark v0 client as deprecated

### CODE REVIEW

File: examples/v0/write.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# v0 API is deprecated
```
Comment (jzelinskie) [inline examples/v0/write.py:1]: Does this work for module docs? I've always seen/used the triple quotes here.

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -50,7 +50,6 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.6"
```
Comment (jzelinskie) [inline .github/workflows/test.yaml:53]: I'm actually hesitant to drop 3.6 until I see the numbers reported for usage.

File: examples/v0/write.py (hunk 1 above)
Comment (samkim) [reply examples/v0/write.py]: Changed these to docstrings. Originally intended these to just be notes to the user but might as well make them follow convention.

**Review by samkim — COMMENTED** [review]
3.6 support added back

**Review by jzelinskie — APPROVED** [review]

**Pull request merged**